grails-app/services/org/pih/warehouse/report/ReportService.groovy

```diff
@@ -688,12 +692,36 @@ class ReportService implements ApplicationContextAware {
         return data
     }
 
+    Map getQuantityOnOrder (String locationId) {
+        def location = Location.get(locationId)
+        def items = orderService.getPendingInboundOrderItems(location)
```

Comment:
Add an option to `getPendingInboundOrderItems` by location and list of products (there is already by location and product)

Reply:
Ok, done